src/biogeochem/CNPhenologyMod.F90

```diff
@@ -143,6 +143,9 @@ module CNPhenologyMod
   logical,  public :: generate_crop_gdds = .false. ! If true, harvest the day before next sowing
   logical,  public :: use_mxmat = .true.           ! If true, ignore crop maximum growing season length
 
+  ! For use with adapt_cropcal_rx_cultivar_gdds .true.
+  real(r8), parameter :: min_gdd20_baseline = 0._r8  ! If gdd20_baseline_patch is ≤ this, do not consider baseline.
```

Comment:
@samsrabin was considering adding this as a namelist option. I recommended:
- adding to the params file OR
- not adding at all

I think @samsrabin prefers the latter because he will converge on some value that is unlikely to ever need updating.